third_party/lua/README-luadebug.md

```diff
@@ -83,7 +89,39 @@ If you have used other CLI debuggers, debugger.lua shouldn't be surprising. I di
     w|where $linecount
     - print source code around the current line
 
-If you've never used a command line debugger before, start a nice warm cozy fire, run tutorial.lua, and open it up in your favorite editor so you can follow along.
+If you've never used a command line debugger before, start a nice warm cozy fire, run luadebug_tutorial.lua, and open it up in your favorite editor so you can follow along.
+
+Breakpoints
+-
+
+One could set breakpoint either using full syntax as:
+
+  break debug-target.lua:9
+
+Or using "short" syntax using only line numbers in a form `+NN` or `:NN`. In this
+case debugger will use current active script file as a basic for such breakpoint.
+
+  b +9
+
+Both those syntaxes mentioned above are equivalent and should behave identically.
+
+If breakpoint(s) saved then execution (command 'continue') will be slowed down a
```

Comment:
`If breakpoint(s) saved, then execution (command 'continue') will be slowed down a bit by calling debugger line hook at each executed line.`

Reply:
Fixed

Reply:
Nit: I still think that "slowed down by **a call** to debugger" is the correct grammar for this sentence.

Reply:
Skipped this one.